src/test/java/io/reactivex/rxjava3/internal/operators/flowable/FlowableOnBackpressureReduceWithTest.java

```diff
@@ -90,6 +90,30 @@ public void simpleBackpressure() {
         ts.assertNotComplete();
     }
 
+    @Test
+    public void reduceBackpressuredSync() {
```

Comment:
what is the difference from `synchronousDrop` test, maybe just rename it to `reduceBacpressuredSync`?

Reply:
Different request pattern.